src/helpers/billing.js

```diff
@@ -188,3 +188,7 @@ export function stripImmediatePlanChange(search) {
   const { immediatePlanChange: _immediatePlanChange, ...options } = qs.parse(search);
   return qs.stringify(options);
 }
+
+export function isProductOnSubscription(state, productName = 'recipient_validation') {
```

Comment:
It might be good to keep the function formatted similar to how they are in `src/helpers/conditions/account` for ease of use. 

```suggestion
  export const isProductOnSubscription = (productName = 'recipient_validation') => state =>
    _.find(_.get(state_.find(_.get(state, 'billing.subscription.products') || {}, { product: productName });
```

and that would change the usage of it to be
```
isRVonSubscription: isProductOnSubscription('recipient_validation')
```

if it's inside a mapStateToProps